src/downloadlistview.h

```diff
@@ -42,14 +42,15 @@ class DownloadProgressDelegate : public QStyledItemDelegate
   Q_OBJECT
 
 public:
-  DownloadProgressDelegate(DownloadManager* manager, DownloadListView* list);
+  DownloadProgressDelegate(DownloadManager* manager, DownloadListView* list, DownloadList* sourceModel);
 
   void paint(QPainter* painter, const QStyleOptionViewItem& option,
              const QModelIndex& index) const override;
 
 private:
-  DownloadManager* m_Manager;
-  DownloadListView* m_List;
+  DownloadManager* m_Manager{nullptr};
```

Comment:
These default values have no real purpose.

Reply:
Removed the defaults